client/lib/wp/sync-handler/cache-index.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import ms from 'ms';
+
+import negate from 'lodash/negate';
+import matchesProperty from 'lodash/matchesProperty';
+import filter from 'lodash/filter';
+import difference from 'lodash/difference';
+
+/**
+ * Internal dependencies
+ */
+import { getLocalForage } from 'lib/localforage';
+
+/**
+ * Module variables
+ */
+const localforage = getLocalForage();
+const debug = debugFactory( 'calypso:sync-handler:cache' );
+
+const debugPromise = ( ...args ) => {
+	return new Promise( resolve => {
```

Comment:
Pretty minor, but since this resolves immediately, we can return a simple function rather than a promise.